pkg/kine/server/otel.go

```diff
@@ -0,0 +1,30 @@
+package server
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/metric"
+)
+
+const name = "k8s-dqlite"
+
+var (
+	tracer    = otel.Tracer(name)
+	meter     = otel.Meter(name)
+	getCnt    metric.Int64Counter
+	listCnt   metric.Int64Counter
+	countCnt  metric.Int64Counter
+	createCnt metric.Int64Counter
+	deleteCnt metric.Int64Counter
+	updateCnt metric.Int64Counter
+)
+
+func init() {
+	listCnt, _ = meter.Int64Counter(fmt.Sprintf("%s.list", name), metric.WithDescription("Number of list requests"))
```

Comment:
Same as in the other package: an error here will cause the code to fail down the line. We should fail here.

As a second thing: could we move each counter closer to where it's used? LIke `listCnt` in the `list.go` file etc etc..